apache/superset
# chore(explore): Get Explore data from endpoint instead of bootstrap_data

### SUMMARY
This PR detaches Explore page from bootstrap data.
Getting `slice`, `form_data` and `datasource` is done by calling the new `/v1/explore` endpoint.
This is a prerequisite to Explore SPA project.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
Everything should work like before (though a bit slower until we finish the rest of Explore work).
Make sure to test different initial urls (go to explore from charts list, dashboard, permalink...)

### ADDITIONAL INFORMATION
